hbase-server/src/test/java/org/apache/hadoop/hbase/client/TestFromClientSide.java

```diff
@@ -1234,7 +1246,14 @@ public void testNull() throws Exception {
       } catch (Exception e) {
         throw new IOException("Using a row with null qualifier threw exception, should ");
```

Comment:
Not your fault but since this is a refactoring, let's fix the message?

Reply:
Sure, fixed.